util/util.js

```diff
@@ -119,15 +119,26 @@ function exitRoom(lobby, roomId, playerId) {
     var room = lobby.getRoomById(roomId)
     room.removePlayer(playerId)
     if (room.isEmpty()) {
+        console.log('Deleting room, since there is no players left in the room')
```

Comment:
Could you specify which room is being deleted? Might be a more useful log message. (Same for a couple of your other log messages like in lines 55 and 85)